Core/Src/ThermistorMonitor.c

```diff
@@ -6,53 +6,149 @@
 
 extern ADC_HandleTypeDef hadc1;
 
+
 // Use NTC 10K Thermistor Parameters
+// Values found here: https://www.vishay.com/docs/29218/ntcle350e4.pdf
+
 #define CALIBRATION_TEMPERATURE 298.15
 #define CALIBRATION_RESISTANCE 10000
-#define DIVIDER_RESISTANCE 10000
 #define THERMISTOR_BETA 3435
+
+#define DIVIDER_RESISTANCE 10000
 #define ADC_RESOLUTION 4095
+#define REFERENCE_VOLTAGE 3.3
+
+// Use Steinhart & Hart coefficients for NTC 10k Thermistor
+// Values found using this site (ordering code: NTCLE350E4103FLB0): https://www.vishay.com/en/thermistors/ntc-rt-calculator/
+// (VALUES ARE HIDDEN UNDER INSPECT ELEMENT)
+#define TEMP_COEFF_A 3.354016434680530000E-3
+#define TEMP_COEFF_B 3.00130825115663000E-4
+#define TEMP_COEFF_C 5.085164943790940E-6
+#define TEMP_COEFF_D 2.18765049258341E-7
+
+//#define RES_COEFF_A -12.89228328
+//#define RES_COEFF_B 4245.14800000
+//#define RES_COEFF_C -87493.00000000
+//#define RES_COEFF_D -9588114.00000000
+
+
 
 ThermistorData_Struct ThermistorData = { .thermistors = {} };
 
+void adcChangeMUX(int channel) {
+	ADC_ChannelConfTypeDef sConfig = {0};
+
+	// Selects new ADC channel for conversion
+	if (DEV_BOARD == 0) {
```

Comment:
```suggestion
	if (!DEV_BOARD) {
```
Consider either an IFDEF here or just writing it a little more clearly